storage/innobase/fil/fil0crypt.cc

```diff
@@ -2525,8 +2535,8 @@ void fil_crypt_thread() {
 
   mutex_enter(&fil_crypt_threads_mutex);
   uint thread_no = srv_n_fil_crypt_threads_started;
-  srv_n_fil_crypt_threads_started++;
   srv_threads.m_encryption_threads_active = true;
+  srv_n_fil_crypt_threads_started++;
```

Comment:
Do we need `m_encryption_threads_active` at all. It looks like it is always `true` when `srv_n_fil_crypt_threads_started > 0`?

Reply:
yes, but this is part of srv_threads structure which was a common place to hold information on active threads and was consulted by the server to check whether all background threads are done executing.

Reply:
Can `srv_n_fil_crypt_threads_started` be moved into `srv_threads` then?

Reply:
done

Reply:
@robgolebiowski The only reason I started this was to avoid unsync states between `m_encryption_threads_active` and `srv_n_fil_crypt_threads_started`. Moving `srv_n_fil_crypt_threads_started` into `srv_threads` is fine but my primary comment was about getting rid of `m_encryption_threads_active`.

Reply:
sorry, I just run sed without thinking - right, I will remove the active - working on it

Reply:
done